cmd/server/assets/users/show.html

```diff
@@ -37,15 +38,67 @@ <h1>{{$user.Name}}</h1>
           {{$user.Email}}
         </div>
 
+        {{if $user.Admin}}
+        <strong>System admin</strong>
+        <div class="card-text text-success mb-3">Enabled</div>
+        {{end}}
+
         <strong>Admin</strong>
         <div class="mb-3">
-          {{$user.CanAdminRealm $currentRealm.ID}}
+          {{if $user.CanAdminRealm $currentRealm.ID}}
+          <div class="card-text text-success mb-3">Enabled</div>
+          {{else}}
+          <div class="card-text mb-3">Disabled</div>
+          {{end}}
         </div>
 
         <a href="/users/{{$user.ID}}/reset-password" data-method="POST" class="btn btn-primary btn-block">Send password reset</a>
       </div>
     </div>
 
+    {{if $currentUser.Admin}}
```

Comment:
I'm -1 on including admin-related actions outside of the admin console. I think we should only have these pages for realm admins and system admin operations need to take place on the system admin console.